webvtt.go

```diff
@@ -59,11 +59,15 @@ func ReadFromWebVTT(i io.Reader) (o *Subtitles, err error) {
 	var item = &Item{}
 	var blockName string
 	var comments []string
+	var indexContent []int
+
```

Comment:
Can you remove this empty line ?

Reply:
fixed with https://github.com/asticode/go-astisub/pull/22/commits/489f53adc2376337082e86637e00f39a74a929b2